submit/submit_online.bats

```diff
@@ -41,7 +42,7 @@ setup() {
 }
 
 @test "binary file emits warning" {
-    cp ./submit $BATS_TMPDIR/binary.c
+    cp $(which bash) $BATS_TMPDIR/binary.c
```

Comment:
Too bad this nice self referencing hack now disappears :-)

Reply:
Yeah. I first tried using bats but that is not a binary 😅